dms/controllers/portal.py

```diff
@@ -16,7 +16,7 @@
 class CustomerPortal(CustomerPortal):
     def _dms_check_access(self, model, res_id, access_token=None):
         try:
-            item_sudo = request.env[model].sudo().browse([res_id])
+            item_sudo = request.env[model].browse([res_id])
```

Comment:
The variable name `item_sudo` is now misleading:

```suggestion
            item = request.env[model].browse(res_id)
```

Change usages below too, of course.

---

Kudos for removing some `sudo`s!